app-tasks/rf/src/rf/utils/cog.py

```diff
@@ -33,7 +34,7 @@ def convert_to_cog(tif_with_overviews_path, local_dir):
     out_path = os.path.join(local_dir, 'cog.tif')
     cog_command = [
         'gdal_translate', tif_with_overviews_path, '-co', 'TILED=YES', '-co',
-        'COMPRESS=LZW', '-co', 'COPY_SRC_OVERVIEWS=YES', '-co', 'BIGTIFF=YES',
+        'COMPRESS=LZW', '-co', 'COPY_SRC_OVERVIEWS=YES', '-co', 'BIGTIFF=IF_SAFER',
```

Comment:
@lossyrob suggested the other day that geotrellis would be faster at decompressing `DEFLATE` so I think we might as well take this opportunity to do that in this PR as well